Dockerfile

```diff
@@ -43,8 +44,6 @@ RUN apt-get update && apt-get install --no-install-recommends --yes nginx
 # Import code, build assets and mirror list
 RUN rm -rf package.json yarn.lock .babelrc webpack.config.js Gemfile.lock nginx.conf
```

Comment:
Hold on, this whole line can be removed right? We're not copying any code into this image up to this point. It all happens below, and none of those files need removing 'cos they're all built files. The same will presumably apply to other eleventy projects.